server/services/premisesService.ts

```diff
@@ -30,6 +33,42 @@ export default class PremisesService {
     return { name: premises.name, summaryList }
   }
 
+  async getOvercapacityMessage(token: string, premisesId: string): Promise<string[] | string> {
+    const premisesClient = this.premisesClientFactory(token)
+    const premisesDateCapacities = await premisesClient.capacity(premisesId)
+    const overcapacityDateRanges = getDateRangesWithNegativeBeds(premisesDateCapacities)
+
+    const overcapacityMessage = this.generateOvercapacityMessage(overcapacityDateRanges)
+    return overcapacityMessage ? [overcapacityMessage] : ''
+  }
+
+  /* istanbul ignore next */
```

Comment:
Why are we ignoring coverage for this?

Reply:
I thought it was causing the test coverage to fail but it seems to be fine now